package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@antoinepairet/icc-api",
-  "version": "1.3.0",
+  "name": "icc-api",
```

Comment:
As long as we do not publish on NPM on our side: OK, we either use the NPM dependency as is or point to a git commit